internal/storage/local/local.go

```diff
@@ -0,0 +1,130 @@
+package local
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	strg "github.com/offen/docker-volume-backup/internal/storage"
+	t "github.com/offen/docker-volume-backup/internal/types"
+	u "github.com/offen/docker-volume-backup/internal/utilities"
+	"github.com/sirupsen/logrus"
+)
+
+type LocalStorage struct {
+	*strg.StorageBackend
+}
+
+// Specific init procedure for the local storage provider.
+func InitLocal(c *t.Config, l *logrus.Logger, s *t.Stats) *strg.StorageBackend {
```

Comment:
So the Go convention is to call such a "constructor" function `NewWhatever`.

if we also apply all of the other conventions Go has for naming things (don't stutter and so on), how about naming this `NewStorageBackend` which other packages would then call as `local.NewStorageBackend`?